.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Bug report
+about: Report a bug for "kn admin"
+title: ''
+labels: kind/bug
+assignees: ''
+
+---
+<!-- If you need to report a security issue with Knative, send an email to knative-security@googlegroups.com. -->
+
+### Bug report
+
+<!-- Please describe what is actually happening -->
+
+### Expected behavior
+
+<!-- Please describe what you expect to happen -->
+
+### Steps to reproduce the problem
+
+<!-- How can a maintainer reproduce this issue (please be detailed) -->
+
+### kn admin version
+
+<!-- Please paste the output of 'kn admin version' in the code block below -->
+
+```
+
+```
+
+### Knative (serving/eventing/client) version
+
+<!-- Remove all except the known affected versions of Knative running on the cluster on which you have detected the issue -->
+
+> Nightly
+> 0.12.x
+> 0.11.x
+> 0.10.x
+> 0.9.x
```

Comment:
I think you can update this list, starting with 0.21 (0.12 is already super old and not supported anymore)

Reply:
Got it, thanks!